tests/gtests/test_SW_VegProd.cc

```diff
@@ -157,11 +157,19 @@ namespace {
 
   // Test summing values across vegetation types
   TEST(VegTest, Summing) {

```

Comment:
In "VegTest, Summing" this now uses the test globals `SW_All.Site.transp_coeff`. If we do that then we need to appropriately reset the globals by calling `Reset_SOILWAT2_after_UnitTest();` at the end of the test.